.travis.yml

```diff
@@ -6,11 +6,13 @@ env:
   - MODE=main
 install:
     # Install runtime
-  - |-
-    sudo sh -c 'echo "deb [arch=amd64] https://apt-mo.trafficmanager.net/repos/dotnet/ trusty main" > /etc/apt/sources.list.d/dotnetdev.list' 
-    sudo apt-key adv --keyserver apt-mo.trafficmanager.net --recv-keys 417A0893
-    sudo apt-get update
-    sudo apt-get install dotnet-dev-1.0.0-rc4-004769 -y
+  - rm -rf ~/.nvm && git clone https://github.com/creationix/nvm.git ~/.nvm && (cd ~/.nvm && git checkout `git describe --abbrev=0 --tags`) && source ~/.nvm/nvm.sh && nvm install 7.10.0
+  - npm install -g npm@'>=5.4.0'
+  - curl https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor > microsoft.gpg
```

Comment:
you may want to add a comment on why these lines are needed and maybe a pointer to where you got the pre-reqs from